src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -46,15 +46,15 @@ An example description is `Centrapay NZD`, `Bitcoin`, or `Ethereum`.
 
 ### Supported Currencies
 
-Payment Requests have a value determined by the currency a Merchant accepts. You should supply a finite list of three-letter [ISO currency codes](https://en.wikipedia.org/wiki/ISO_4217).
+[Payment Requests](https://docs.centrapay.com/api/payment-requests) have a value determined by the currency a Merchant accepts. You should supply a finite list of three-letter [ISO currency codes](https://en.wikipedia.org/wiki/ISO_4217) that is supported by your asset.
 
-### Example Payment Option
+### Example Defintiation
```

Comment:
I think this change may have been unintentional.

Reply:
Yes, thanks :)

Reply:
It was intentional. The H2 is called Defining a payment method, so thought it was redundant to add Payment Method to this header